coordinator/src/main/scala/com/socrata/datacoordinator/resources/collocation/CollocationManifestsResource.scala

```diff
@@ -41,4 +46,21 @@ case class CollocationManifestsResource(instanceId: Option[String],
       }
     }
   }
+
+  private def dropCollocations(req: HttpRequest): HttpResponse = {
+    (instanceId, datasetId.flatMap(DatasetInternalName(_))) match {
+      case (Some(instance), Some(dataset)) =>
+        coordinator.dropCollocationsOnInstance(instance, dataset) match {
+          case None => OK
+          case Some(InstanceNotFound(_)) => instanceNotFound(instance)
+          case Some(_) => InternalServerError
+        }
+      case (None, _) => instanceNotFound("")
+      case (_, None) => errorResponse(
+        BadRequest,
```

Comment:
Isn't internal name not found an internal error?

Reply:
Not to data-coordinator? It would be an internal error to soda-fountain I think.

Reply:
ok